src/main.rs

```diff
@@ -347,32 +354,37 @@ impl Kopium {
     }
 
     fn print_prelude(&self, results: &[Container]) {
-        if !self.hide_kube {
+        if !self.hide_kube && !self.no_import.contains(&Import::Kube) {
             println!("use kube::CustomResource;");
         }
-        if self.builders {
+        if self.builders && !self.no_import.contains(&Import::Builder) {
             println!("use typed_builder::TypedBuilder;");
         }
-        if self.derive.contains(&"JsonSchema".to_string()) {
+        if self.derive.contains(&"JsonSchema".to_string()) && !self.no_import.contains(&Import::Schemars) {
             println!("use schemars::JsonSchema;");
         }
-        println!("use serde::{{Serialize, Deserialize}};");
-        if results.iter().any(|o| o.uses_btreemaps()) {
+        if !self.no_import.contains(&Import::Serde) {
+            println!("use serde::{{Serialize, Deserialize}};");
+        }
+        if results.iter().any(|o| o.uses_btreemaps()) && !self.no_import.contains(&Import::Maps) {
             println!("use std::collections::BTreeMap;");
         }
-        if results.iter().any(|o| o.uses_hashmaps()) {
+        if results.iter().any(|o| o.uses_hashmaps()) && !self.no_import.contains(&Import::Maps) {
             println!("use std::collections::HashMap;");
         }
```

Comment:
This is potentially contentious; re-using `Import::Maps` for both map types because we are meant to be consistent with map usage (and we now have a `--map-type` flag to switch between them).

Am also doing the same for `Import::Chrono` below because each use imports potentially multiple structs and didn't want to make `3` cases just for chrono.